src/main/java/picard/arrays/illumina/InfiniumGTCFile.java

```diff
@@ -668,6 +668,26 @@ public int getAbCalls() {
         return abCalls;
     }
 
+    public int[] getRawXIntensities() { return rawXIntensities; }
```

Comment:
The code here is inconsistent but I think formatting across lines is better and follows the oracle/sun standard. Most IDEs can collapse them anyway so they look like a single line when you're editing the code. So please reformat the new getters:
```
    public int[] getRawXIntensities() {
        return rawXIntensities;
    }
```

Reply:
That was just me manually collapsing them.  I couldn't distinguish it from the IDE's autocollapse.